core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -130,20 +131,20 @@ export class Question {
     if (!atLeastOneAnswerCorrect) {
       return 'At least one answer should be marked correct';
     }
-    return null;
+    return '';
   }
 
   getUnaddressedMisconceptionNames(misconceptionsBySkill: {}): string[] {
```

Comment:
You will need to change the type of `misconceptionsBySkill` so that it can be indexed by string I think.

Reply:
type of `misconceptionsBySkill`  was mentioned as `Misconception[]` in question-validation.service.ts, so I used the same, but it is showing some errors while taking string as index. I am not able to figure out the right type for `misconceptionsBySkill`, please help me out here.
Thanks @vojtechjelinek

Reply:
You will need to analyze the code, both in this function and in the places where this function is called to understand what is going on and what should the proper type be.